CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log - AWS SDK for Android
 
+## [Release 2.16.1](https://github.com/aws/aws-sdk-android/releases/tag/release_v2.16.1)
+
+### New Features
```

Comment:
Minor: Change this to `Misc. Updates` as we used to do dependency version updates through `Misc. Updates` for AppSync SDKs.